pkgs/development/node-packages/default.nix

```diff
@@ -67,38 +67,34 @@ let
       '';
     };
 
-    coc-imselect = super.coc-imselect.override {
-      meta.broken = since "10";
-    };
+    coc-imselect = super.coc-imselect.override (oldAttrs: {
+      meta = oldAttrs.meta // { broken = since "10"; };
+    });
 
-    dat = super.dat.override {
+    dat = super.dat.override (oldAttrs: {
       buildInputs = [ self.node-gyp-build pkgs.libtool pkgs.autoconf pkgs.automake ];
-      meta.broken = since "12";
-    };
+      meta = oldAttrs.meta // { broken = since "12"; };
+    });
 
-    deltachat-desktop = super."deltachat-desktop-../../applications/networking/instant-messengers/deltachat-desktop".override {
-      meta.broken = true; # use the top-level package instead
-    };
+    deltachat-desktop = super."deltachat-desktop-../../applications/networking/instant-messengers/deltachat-desktop".override (oldAttrs: {
+      meta = oldAttrs.meta // { broken = true; }; # use the top-level package instead
+    });
 
     # NOTE: this is a stub package to fetch npm dependencies for
     # ../../applications/video/epgstation
-    epgstation = super."epgstation-../../applications/video/epgstation".override (drv: {
-      meta = drv.meta // {
-        platforms = pkgs.lib.platforms.none;
-      };
+    epgstation = super."epgstation-../../applications/video/epgstation".override (oldAttrs: {
+      meta = oldAttrs.meta // { platforms = pkgs.lib.platforms.none; };
```

Comment:
I don't understand this line.

Reply:
I don't understand it either. This was the original (all I did was change `drv` to `oldAttrs` put it all on one line):

```nix
    epgstation = super."epgstation-../../applications/video/epgstation".override (drv: {
      meta = drv.meta // {
        platforms = pkgs.lib.platforms.none;
      };
    });
```

Reply:
Probably related to comment on L102-103